src/Calculator/Views/MainPage.xaml.cs

```diff
@@ -604,6 +604,11 @@ private static void SetSettingsButtonAccessKey(MUXC.NavigationView navView)
             }
         }
 
+        private double NavigationViewOpenPaneLength(bool isAlwaysOnTop)
+        {
+            return isAlwaysOnTop ? 0 : 256;
```

Comment:
I suggest we still have the panel length defined in App.xaml, and here read it by  `Application.Current.Resources["SplitViewOpenPaneLength"])` .

Reply:
Done.